projectdo

```diff
@@ -287,6 +287,14 @@ try_latex() {
   fi
 }
 
+# .NET
+
+try_dotnet() {
+  if [ -f *.csproj ] || [ -f *.fsproj ] || [ -f *.sln ]; then
```

Comment:
Shellcheck [complains about this](https://www.shellcheck.net/wiki/SC2144) and it does indeed fail if there is more than one file with one of these extensions in the same directory. For instance, I tried adding a second `.sln` file in `tests/dotnet-sln/` and then the tests fail. This is probably not likely to happen, but it would be nice to handle this case as well for robustness.

One solution would be to use `find` like this:
```sh
if [ -n "$(find . -maxdepth 1 \( -name '*.csproj' -o -name '*.fsproj' -o -name '*.sln' \) -print -quit)" ]; then
```

Reply:
You're right, I'll take this solution. Thanks!